lib/facilities/drivetime_bands/response.rb

```diff
@@ -0,0 +1,23 @@
+# frozen_string_literal: true
+
+require 'common/models/base'
+
+module Facilities
+  module DrivetimeBands
+    class Response < Common::Base
+      attribute :body, String
+
+      def initialize(body)
+        self.body = body
```

Comment:
You could move the json parse step to this initialize function and store the json as another instance variable. I think that would simplify some of the code in the client